project/Http4sBuild.scala

```diff
@@ -53,12 +53,8 @@ object Http4sBuild {
     }
 
   lazy val alpnBoot            = "org.mortbay.jetty.alpn"    % "alpn-boot"               % "8.1.9.v20160720"
-  def argonaut(scalazVersion: String) = "io.argonaut"       %% "argonaut"                % scalazCrossBuild("6.1", scalazVersion)
-  def argonautShapeless(scalazVersion: String) = "com.github.alexarchambault" %% "argonaut-shapeless_6.1" %
-    (VersionNumber(scalazVersion).numbers match {
-      case Seq(7, 1, _*) => "1.1.0"
-      case Seq(7, 2, _*) => "1.1.1"
-    })
+  lazy val argonaut            = "io.argonaut"              %% "argonaut"                % "6.2-M3"
```

Comment:
We upgraded to argonaut-6.2 in an earlier version of http4s, and the community revolted because it wasn't ready yet.  But 6.2 breaks the scalaz dependency in argonaut, which makes sense for this branch.

I _think_ this is the proper time to make this upgrade, but I'm going to leave this one open a bit longer to see if anybody disagrees.